src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -176,6 +178,27 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
 
   const totalPages = callPages.length;
 
+  const isSpaceOrEnterKey = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    return event.key === KEY.ENTER || event.key === KEY.SPACE;
```

Comment:
Why do you not use the helper functions in the KeyboardUtil (like `isSpaceKey`)?

Reply:
Removed this anyway as it's not necessary here. About KeyboardUtill there we are using a regular `KeyboardEvent` as arguments and didn't want to mix it up.